e2e/cypress/integration/channel/header_spec.js

```diff
@@ -81,6 +81,40 @@ describe('Header', () => {
         // # Verify that the Search term input box is still cleared and search term does not reappear when RHS opens
         cy.get('#searchBox').should('be.visible').and('be.empty');
     });
+
+    describe('Header update', () => {
+        before(() => {
+            // # cheange channel header
+            cy.updateChannelHeader('updated header');
+        });
+
+        beforeEach(() => {
+            // # hover on last message i.e system message
+            cy.get('.post--system').last().trigger('mouseover');
+        });
+
```

Comment:
We generally have only one describe, before and beforeEach methods for each test file.

Reply:
Also please mention the test key and title as mentioned in this ticket https://mattermost.atlassian.net/browse/MM-17458
*Test Key:* M17458
*Title:* System message limited options

Reply:
as discussed changes are done and pushed